client/layout/guided-tours/main-tour.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+
+import { getSelectedSite, isPreviewShowing } from 'state/ui/selectors';
+import { isNewUser } from 'state/ui/guided-tours/selectors';
+import { Tour,
+	Step,
+	Next,
+	Quit,
+	ClickToContinue,
+	Link,
+} from 'layout/guided-tours';
+
+const selectedSiteIsPreviewable = state =>
+	getSelectedSite( state ) && getSelectedSite( state ).is_previewable;
+
+const selectedSiteIsCustomizable = state =>
+	getSelectedSite( state ) && getSelectedSite( state ).is_customizable;
+
+const previewIsNotShowing = state =>
+	! isPreviewShowing( state );
+
+export const MainTour = ( { translate } ) => (
+	<Tour name="main" version="20160601" path="/" context={ isNewUser }>
+		<Step name="init" placement="right" next="my-sites">
+			{
+				translate( "{{strong}}Need a hand?{{/strong}} We'd love to show you around the place," +
+										'and give you some ideas for what to do next.',
+					{
+						components: {
+							strong: <strong />,
+						}
+					} )
+			}
+			<Next>{ translate( "Let's go!" ) }</Next>
+			<Quit>{ translate( 'No thanks.' ) }</Quit>
+		</Step>
+
+		<Step name="my-sites"
+			target="my-sites"
+			placement="below"
+			arrow="top-left"
+			next="sidebar"
+		>
+			{
+				translate( "{{strong}}First things first.{{/strong}} Up here, you'll find tools for managing " +
+										"your site's content and design.",
+					{
+						components: {
+							strong: <strong />,
+						}
+					} )
+			}
+			<ClickToContinue icon="my-sites" target="my-sites" />
```

Comment:
Maybe this could be renamed `Continue`, so we could do `<Continue icon="cross" target="web_preview_close" context={ previewNotShowing }`?